Abcrete/BangazonCLI
# Order manager

## Status
**READY**

## Description
Complete all methods that deal with orders- add a product to an order, create a new order, return all orders, add payment to an order, return revenue report for a customer.


## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout orderManager
3. in test folder, "dotnet test"
4. confirm tests pass

## Impacted Areas in Application
Order.cs
OrderManager.cs
DatabaseInterface.cs

## Link to Feature Ticket
#10 
#13 
#6
